flask_api/app.py

```diff
@@ -29,14 +36,54 @@ def parse_user_id(req):
     print(req.headers)
     return req.headers['Authorization'].split()[1]
 
+def resize_n_upload(original_file, New_file, date_time, s3_key_name):
+    s3_client = boto3.client('s3')
+    s3_bucket_name = "image-bucket-paul"
+
+    im = Image.open(original_file)
+
+    if New_file == '/tmp/thumb_electricals.jpeg':
+        im = im.filter(filter=ImageFilter.BLUR)
+        im = im.transpose(Image.ROTATE_90)
+        size = (480, 640)
+        im.thumbnail(size) 
+    else:
+        size = (1024, 768)
+        im.thumbnail(size) 
+    
+    im.save(New_file)
+
+    s3_client.upload_file(New_file, s3_bucket_name, s3_key_name)
+
 @app.route('/lists')
 def fetch_lists():
     try:
         user_id = parse_user_id(request)
     except:
         return jsonify('Unauthorized'), 401
 
+    if random.uniform(0, 100) > 50 :
+        now = datetime.now() # current date and tim
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The naive datetime objects are treated by many datetime methods as local times, it is preferred to use aware datetimes to represent times in UTC. The recommended way to create an aware datetime object representing a specific timestamp in UTC is by passing `tzinfo` as an argument to the method.

[Learn more](https://docs.python.org/3/library/datetime.html#aware-and-naive-objects)